packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import * as AtomicBreadboxStories from './atomic-breadbox.new.stories';
+import { AtomicDocTemplate } from '../../../../storybook-utils/documentation/atomic-doc-template';
+
+<Meta of={AtomicBreadboxStories} />
+
+<AtomicDocTemplate
+  stories={AtomicBreadboxStories}
+  githubPath="search/atomic-breadbox/atomic-breadbox.ts"
+  tagName="atomic-breadbox"
+  className="AtomicBreadbox"
+>
+
+The `atomic-breadbox` component creates breadcrumbs that display a summary of the currently active facet values.
+This component helps users understand and manage their current filter selections by showing them in a compact, visual format.
+
+This component is typically placed within the "status" section of the layout, above the search results.
+
+```html
```

Comment:
I think the example should focus on the essential (kind of like the atomic-commerce-breadbox).